src/vs/workbench/contrib/scm/electron-browser/scmViewlet.ts

```diff
@@ -884,10 +884,20 @@ export class RepositoryPanel extends ViewletPanel {
 			new ResourceRenderer(this.listLabels, actionItemProvider, () => this.getSelectedResources(), this.themeService, this.menus)
 		];
 
-		this.list = this.instantiationService.createInstance(WorkbenchList, this.listContainer, delegate, renderers, {
-			identityProvider: scmResourceIdentityProvider,
-			keyboardNavigationLabelProvider: scmKeyboardNavigationLabelProvider
-		}) as WorkbenchList<ISCMResourceGroup | ISCMResource>;
+		const horizontalScrollingSCM = this.configurationService.getValue('workbench.tree.horizontalScrollingSCM') || 'inherit';
+
+		if (horizontalScrollingSCM === 'inherit') {
```

Comment:
Instead of the `if { ... } else { ... }` blocks we could simplify this to one expression:

```typescript
this.list = this.instantiationService.createInstance(WorkbenchList, this.listContainer, delegate, renderers, {
    identityProvider: scmResourceIdentityProvider,
    keyboardNavigationLabelProvider: scmKeyboardNavigationLabelProvider,
    horizontalScrolling: horizonalScrollingSCM === 'inherit' ? undefined : horizontalScrollingSCM === 'true'
}) as WorkbenchList<ISCMResourceGroup | ISCMResource>;
```